tree/tree/inc/TTreeCacheUnzip.h

```diff
@@ -29,26 +29,41 @@
 
 #include <queue>
 
+#ifdef R__USE_IMT
+#include "tbb/task.h"
```

Comment:
The tbb includes should be hidden from the header files in any case.  I've just noticed there are some that made it to the master, but they will have to go too.

In your case, the solution may be as easy as forward-declaring  tbb::task. Give it a try.